vamb/__main__.py

```diff
@@ -1248,13 +1249,14 @@ def write_clusters_and_bins(
     clusters: dict[str, set[str]],
     sequence_names: Sequence[str],
     sequence_lens: Sequence[int],
+    to_file: bool = True
 ):
     # Write unsplit clusters to file
     begintime = time.time()
     unsplit_path = Path(base_clusters_name + "_unsplit.tsv")
-    with open(unsplit_path, "w") as file:
+    with open(unsplit_path, "a") as file:
```

Comment:
I'm not too wild about the idea of opening and closing a file for every cluster written. This takes time, especially on a compute cluster where the file system is slow. I suggest instead to refactor the code such that all clustering output files are kept open during the clustering process.